plugins/module_utils/entity.py

```diff
@@ -157,3 +157,15 @@ def _fetch_url(self, url, method, data=None, timeout=30):
             )
         status = {"error": err, "code": status_code}
         return resp_json, status
+
+    def get_intersection_of_spec(self, spec1, spec2):
```

Comment:
Change the name to get_combined_spec() or get_unified_spec() or say, unify_spec() or combine_spec()

Reply:
Done